src/exc2/Visualisation.py

```diff
@@ -0,0 +1,36 @@
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy as np
+import time
+import math
+plt.style.use('ggplot')
+
+# arrays to accumulate time and function results
+current_array = []
+func_array = []
+displayed_data = 20
+
+# the time when the script started
+start_time = time.time()
+# signal to terminate the script if the plot was closed
+go_signal = True
+
+while True and go_signal:
```

Comment:
```suggestion
while go_signal:
```

The `while True` part becomes redundant if you already have the `go_signal` part, right? :smile: